src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -769,38 +747,125 @@ class ContentSidebar extends PureComponent<Props, State> {
         });
     };
 
+    /**
+     * Sorts and formats assignments, and then adds them to their task. 
+     *
+
+     * @param {Task} task - Task to which the assignments belong
+     * @param {Task} assignments - List of task assignments
+     * @return {Task}
+     */
+    appendAssignmentsToTask(task: Task, assignments: Array<TaskAssignment>) {
```

Comment:
This would be good to put in AF API

Reply:
it would indeed, or would be great to just specify this format with graphQL :)